tests/unit/lfp/lfp.cpp

```diff
@@ -0,0 +1,104 @@
+#define BOOST_TEST_MODULE LFPTest
+#define BOOST_TEST_MAIN
+
+#include <iostream>
+
+#include <boost/test/unit_test.hpp>
+#include <boost/test/test_case_template.hpp>
+
+#include "coreneuron/io/lfp.hpp"
+#include "coreneuron/mpi/nrnmpi.h"
+
+template <typename F>
+double integral(F f, double a, double b, int n) {
```

Comment:
~instead of implementing the numerical integral here we could use boost. For example:~
~https://www.boost.org/doc/libs/1_66_0/libs/math/doc/html/math_toolkit/trapezoidal.html~

EDIT: ok, they told me that we do not really have boost in coreneuron. It is used only for testing

Reply:
Ok.